.github/codeql-queries/PotentiallyDangerousFunction.ql

```diff
@@ -42,7 +42,7 @@ predicate potentiallyDangerousFunction(Function f, string message) {
     message = "Call to ntohs() is confusing. Use be16toh() instead."
   ) or (
     f.getQualifiedName() = "strerror" and
-    message = "Call to strerror() is not thread-safe. Use strerror_r() or printf()'s %m format string instead."
+    message = "Call to strerror() is not thread-safe. Use STRERROR() or printf()'s %m format string instead."
```

Comment:
maybe swap the order of the suggestions in the second sentence. %m is probably a lot nicer than STRERROR() in most cases.